packages/react-reconciler/src/__tests__/ReactLazy-test.internal.js

```diff
@@ -213,14 +213,18 @@ describe('ReactLazy', () => {
       unstable_isConcurrent: true,
     });
 
+    function App() {
+      return (
+        <Suspense fallback={<Text text="Loading..." />}>
+          <LazyText text="Hi" />
+        </Suspense>
+      );
+    }
+
     let error;
     try {
       await act(() => {
-        root.update(
-          <Suspense fallback={<Text text="Loading..." />}>
-            <LazyText text="Hi" />
```

Comment:
We have a lot of test like this that execute at the top level and so has no owner. Therefore doesn't get any owner stack which means that it doesn't have a component stack which fails our assertions. This is why I usually try to add a single frame if possible. Mainly due to our tests. This was relying on the top stack to be able to have a single stack frame at all.

We could maybe also add a "Lazy" frame if there's no other owner.